kb/plane_geometry/formal_statements/statements/statements_scs/stat_obtuse_triangle_height_side_straight_line.scs

```diff
@@ -0,0 +1,14 @@
+stat_obtuse_triangle_height_side_straight_line <- statement;;
+
+stat_obtuse_triangle_height_side_straight_line -> rrel_key_sc_element: concept_obtuse_triangle; nrel_height; nrel_side; concept_straight_line;;
+
+stat_obtuse_triangle_height_side_straight_line => nrel_main_idtf: [Акс.(тупоугольный треугольник;высота*;сторона*;прямая)] (*<- lang_ru;;*);;
+stat_obtuse_triangle_height_side_straight_line => nrel_main_idtf: [Ax.(obtuse triangle;height*;side*;straight line)] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]